demisto_sdk/commands/validate/validators/PA_validators/PA114_pack_metadata_version_should_be_raised.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from packaging.version import Version
+
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Pack
+
+
+class PackMetadataVersionShouldBeRaisedValidator(BaseValidator[ContentTypes]):
+    error_code = "PA114"
+    description = "Ensure that the pack metadata is raised on relevant changes."
+    rationale = (
+        "When updating a pack, its version needs to be raised to maintain traceability."
+    )
+    error_message = (
+        "The pack version (currently: {old_version}) needs to be raised - "
+        "make sure you are merged from master and "
+        'update the "currentVersion" field in the '
+        "pack_metadata.json or in case release notes are required run:\n"
+        "`demisto-sdk update-release-notes -i Packs/{pack} -u "
+        "(major|minor|revision|documentation)` to "
+        "generate them according to the new standard."
+    )
+    related_field = "currentVersion, name"
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
```

Comment:
I don't see any reference to the type of change here at all seems like we always require bump based on this logic